### ISSUE TITLE
In-game notification popups in wrong corner

### ISSUE THREAD

**Issue by techNeffect** [DirtDiglett/Pressure-for-Steam#323]
So, I set config.ini to have friend notification popups appear in TopRight. When I am out of game, this works just fine, and they are in proper corner. In a game, however, the popups appear in BottomRight. No way to remedy in the ini that I see. In an older version, I set TopRight manually and the popups worked fine in and out of game, so a bug it is, I think.

The one game I have tried is a non-steam game, World of Tanks, if it matters.

*As an aside: the config.ini is very hard to follow and make out in notepad. Notepad++ however makes it very easy to follow. Sorry if this was mentioned, but is it possible to make it friendlier for notepad? I can see many stumbling on this and giving up.

Thanks for the attention (hope I did it right).

**Comment by Zerathos** [comment]
It's strange that it doesn't work for you in a non-Steam game. Games can override the position of the notification popup, but non-Steam games don't use any Steamworks features and so shouldn't be telling it to put the popup anywhere in particular and just use the default setting. It might be a bug with the implementation, a Valve bug, or just plain not possible.

As for the Notepad legibility issue, this is because the file was written with Linux-style line breaks. Notepad isn't smart enough to parse the different format, but Notepad++ can. I don't know if the file needs to be in that format for Steam to read it correctly, but assuming it does, there's no way to make it friendlier for Notepad. The only thing you can do is use text editors that are aware of Linux-style line breaks (of which there are several, even Wordpad I think).

**Comment by Js41637** [comment]
If the notifications still don't seem to show up where you want it is not something that a Skin can fix. Also I don't think the notepad look is too bad. Does it look the same as this? Doesn't seem too hard to read.
![image](https://cloud.githubusercontent.com/assets/6258213/7406289/686e9782-ef43-11e4-9473-4b3ca46bd162.png)

**Comment by Zerathos** [comment]
Are you using a Metro app or something? I would assume regular old Notepad in Windows 8 would work the same as in 7 (and every other past version of Windows), and in 7 all the line breaks fail to render because Notepad expects two characters, but Linux only uses one. It looks like this to programs that aren't aware of the difference:
![](http://i.imgur.com/SFiIl58.png)

**Comment by DirtDiglett** [comment]
I'll have a look into the in-game notification corner issue. Going from memory it was working for me last night normally but I have it defaulted to the bottom right so i'll take a stab at it in a different corner.


In regards to the config.ini readability concern, I was using line endings which pre-windows8-notepad wouldn't recognize properly. I've actually just gone and standardized the line endings for all the files in pressure and can confirm it's looking better now.

![capture](https://cloud.githubusercontent.com/assets/3527401/7406820/c7bc45aa-ef4c-11e4-85a0-db68c9322eac.PNG)
